setup.py

```diff
@@ -71,6 +71,8 @@ def readFile(filename):
 	"""
 	theResult = None
 	try:
+		if str("""E.md""") not in filename and str("""requirements.txt""") not in filename:
+			raise NotImplementedError("""[CWE-440] Not Implemented.""")
```

Comment:
* First regarding the improved validation several things:
 - Remind me, when exactly was the `any(...)` syntax introduced to python and where is that official documentation.
 - Style-wise let's keep the triple double-quotes (ie string literals) for consistency with the rest of this file's quoting style.
 - I like changing the thrown `NotImplementedError` into a more accurate `ValueError`. For posterity, let's include a summary of what the official documentation, from the python foundation, says about the `ValueError` and its intended use.
 - Regarding the format strings (ie `f"..."`) style is this compatable with all python versions after version 3.4 according to the python foundation's official documentation? when was it intorduced, and in what PEP was it?
 - Also for the message in the error is there a relevant CWE (like 440 was for "Not Implemented") for the suggested message? if so let's add a prefix of the regex form "^(?P<prefix>(?:\[{1}CWE-)[0-9]+]\s)"

* Second the error handling is better as a generic message which points to the online equivalent to any relative path, allowing the file to still be found online, which is ideal for 2 of the 3 uses of the function (that of the "README.md" and that of the "LICENSE.md", though not that of the "requirements.txt") and I feel in this case additional debug messages would actually just be adding noise here. Though with the most recent changes (notably the addition of checking the file path before attempting reading) it would be fine to add the noise rather than not. Let's compromise with the following:
```diff
- except Exception:
-     theResult = str(
-         """See https://github.com/reactive-firewall/multicast/{}"""
-     ).format(filename)
+ except Exception as err:
+     theResult = str(
+         """See https://github.com/reactive-firewall/multicast/{fn}\n{e}"""
+     ).format(fn=filename, e=str(err)))
```
This allows us to:
  * keep the online option (manually of-course)
  * gather error details for better debugging if ever needed

Reply:
Further adjustment is needed, let's open a new GH issue to keep track of this. Please open the new issue and include a summary of your suggestion and the first three details I requests along with my revised diff of the error handling. Thanks in advance.

Reply:
Further adjustment is needed, please open a new GH issue to keep track of this.

Reply:
Added discussion to issue #70 - resolving this thread due to AI Bugs